package.json

```diff
@@ -198,7 +206,7 @@
 				"sourcery.token": {
 					"type": "string",
 					"default": "",
-					"description": "Sourcery token. You can find your token at https://sourcery.ai/dashboard"
+					"description": "DEPRECATED - Run the \"Sourcery: Login\" command instead"
```

Comment:
In [VS Code's specs](https://code.visualstudio.com/api/references/contribution-points#contributes.configuration), they describe the `deprecationMessage` and `markdownDeprecationMessage` fields which may be exactly what we want:

> If you set `deprecationMessage`, or `markdownDeprecationMessage`, the setting will get a warning underline with your specified message. It won't show up in the settings UI unless it is configured by the user. If you set `markdownDeprecationMessage`, the markdown will not be rendered in the setting hover or the problems view. If you set both properties, `deprecationMessage` will be shown in the hover and the problems view, and `markdownDeprecationMessage` will be rendered as Markdown in the settings UI.

So you should probably keep the old `description` and add this deprecation message instead.

Reply:
Nice!